src/util.js

```diff
@@ -24,6 +24,18 @@ export const numberFormatKeys = [
   'maximumSignificantDigits'
 ]
 
+export const dateTimeFormatKeys = [
+  'weekday',
+  'era',
+  'year',
+  'month',
+  'day',
+  'hour',
+  'minute',
+  'second',
+  'timeZoneName'
+]
```

Comment:
I think this misses these [options ](https://devhints.io/wip/intl-datetime)

timeZone: 'Asia/Shanghai',
  // Force 12-hour or 24-hour
  hour12: true | false,

  // Rarely-used options
  hourCycle: 'h11' | 'h12' | 'h23' | 'h24',
  formatMatcher: 'basic' | 'best fit'